### ISSUE TITLE
False positive against Sentry-Trace

### ISSUE THREAD

**Issue by TomBeckett** [PicnicSupermarket/error-prone-support#315]
### Describe the bug

- [x] I have verified that the issue is reproducible against the latest version
      of the project.
- [x] I have searched through existing issues to verify that this issue is not
      already known.

### Minimal Reproducible Example

```java
    @SentrySpan
    @Transactional
    @Scheduled(cron = "@hourly")
    @SchedulerLock(name = SYNC_EXERCISES_LOCK_NAME, lockAtLeastFor = "2m", lockAtMostFor = "10m")
    @SentryTransaction(operation = "task")
```
Produces:

```bash
[ERROR]     (see https://error-prone.picnic.tech/bugpatterns/ScheduledTransactionTrace)
[ERROR]   Did you mean '@Trace(dispatcher = true)@SentrySpan'?
```

### Expected behavior

No error should be present. We use Sentry (not New Relic).

### Setup

- Mac OS latest.
- Java version 19.0.1
- Error Prone Latest
- Error Prone Support Latest

### Additional context

None.

**Comment by rickie** [comment]
Hey @TomBeckett, thanks for opening this issue!

Good news, we are already working on a fix for that! For context, see #308. There we add an extra check to make sure that an error is thrown only if New Relic is on the classpath. In other words, it should fix your case.   

For now, I would suggest to turn off the check as is explained in our [documentation](https://error-prone.picnic.tech/bugpatterns/ScheduledTransactionTrace/). 

We will let you know once a fix for this is out.

**Comment by TomBeckett** [comment]
@rickie Awesome, thank you for the fast response. Is there any ETA on when this may be released?

I ask because I have the more issues/examples just like that thread and don't want to spam issues if they are coming soon 😄

**Comment by rickie** [comment]
I hope we can do a release ~this week (not making a hard promise though 👀).

Do you mean that you have more _related_ issues or _other_ issues that are not related to this one? 
In the latter case, feel free to open more issues 😄.

**Comment by rickie** [comment]
@TomBeckett we released [v0.5.0]( https://github.com/PicnicSupermarket/error-prone-support/releases/tag/v0.5.0)! So updating should solve the problem 😄. Let us know if you encounter any problem.
